### DISCUSSION TITLE
Stacked or piled cards

### DISCUSSION THREAD

**Discussion by smartm0use** [fomantic/Fomantic-UI#1945] [Views (card, comment, item, statistic,...)] [answered]
Hello, I need [stacked](https://fomantic-ui.com/elements/segment.html#stacked) or [piled](https://fomantic-ui.com/elements/segment.html#piled) effect on cards, as seen for segments.

![image](https://user-images.githubusercontent.com/5399265/114691894-6be6dd00-9d18-11eb-8b0c-f8b56cc674fb.png)

Any way to do that?
Or do you know how to make it better to mean that a folder contains subfolders contrary to a card that doesn't have subfolders?

**Comment by lubber-de** [comment, marked as answer]
Try to combine `card` and `segment` into `ui stacked segment card` or `ui piled segment card`
See https://jsfiddle.net/lubber/dkocr4nw/5/
![image](https://user-images.githubusercontent.com/18379884/114735411-6c499d00-9d45-11eb-8fb3-55ede0b7ebf1.png)

**Comment by smartm0use** [comment]
Very interesting, thank you.
Do you know a way to have more "cards" in the stack that appear under the main card?

**Reply by lubber-de** [reply]
You can use `ui tall stacked segment card` for another layer, but that's the limit without extra css (if you want more layers you have to wrap the card around another stacked card segment
See
https://jsfiddle.net/lubber/dkocr4nw/11/

**Comment by smartm0use** [comment]
I see, ok. I will try to go for "tall" without wrapping more.
Thank you.
